File: nicolaspuebla_proyecto_final/src/main/java/com/example/nicolaspuebla_proyecto_final/model/dto/TrainingCreation.java
package com.example.nicolaspuebla_proyecto_final.model.dto;

/**
 * Clase que contiene la información necesaria para crear un entrenamiento.
 */
public class TrainingCreation extends EventCreation {

    public TrainingCreation(){}

    public TrainingCreation(String event_type, Long teamId, Double latitude, Double longitude, String date) {
        super(event_type, teamId, latitude, longitude, date);
    }
}

File: nicolaspuebla_proyecto_final_android/app/src/main/java/com/example/nicolaspuebla_proyecto_final_android/data/model/dto/TrainingCreation.kt
package com.example.nicolaspuebla_proyecto_final_android.data.model.dto

/**
 * Clase empleada en la petición para la creación de un entrenamiento.
 */
data class TrainingCreation (
    override val event_type: String = "match",
    override val teamId: Long,
    override val latitude: Double,
    override val longitude: Double,
    override val date: String
): EventCreation(event_type, teamId, latitude, longitude, date)
